### PR TITLE
Adding mock triple pedal

### PR DESCRIPTION
**Pull request by ethanfowler**

### CODE REVIEW

File: sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# import getch
```
Comment (mikramarc) [inline sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py]: remove

File: sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        self._kb_listener = None
+        if keyboard_control:
+            self._kb_listener = keyboard.Listener(on_press=self._on_keyboard_press,
+                                                  on_release=self._on_keyboard_release)
+
+    def run(self):
+        if self._kb_listener is not None:
+            self._kb_listener.start()
+            rospy.loginfo("Mock pedal keyboard control enabled.\n" +
+                          "╔════════════╦═════════════════════════╗\n" +
```
Comment (mikramarc) [inline sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py:51]: say whaaa

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py (hunk 1 above)
Comment (ethanfowler) [reply sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py]: Fixed in https://github.com/shadow-robot/sr_teleop_devices/pull/23/commits/0140a9569701e15da01d7bfef4f21687601fa135

File: sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py (hunk 2 above)
Comment (ethanfowler) [reply sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py:51]: Advanced UI design mate

**Review by mikramarc — APPROVED** [review]

**Pull request merged**